thresher/models.py

```diff
@@ -65,11 +65,20 @@ class Topic(models.Model):
     # an id within the given Analysis Type
     topic_id = models.IntegerField() 
 
+    # an id of its parent topic
+    parent_id = models.ForeignKey(self, related_name="subtopics")
+
     # The analysis type to which this topic belongs
-    analysis_type = models.ForeignKey(AnalysisType, related_name='topics')
+    # analysis_type = models.ForeignKey(AnalysisType, related_name='topics')
 
     # The name of the topic
     name = models.TextField()
+
+    # Glossary for analysis, copied from Analysis Type
```

Comment:
Give a better description here, if you know what glossary is supposed to be.  The person reading the code does not care very much where this line came from, rather what it's doing there.